SrcDragonfly/masterParallel.m

```diff
@@ -1,4 +1,4 @@
-function [fOutVar,nBlockPerCPU, totCPU] = masterParallel(Parallel,fBlock,nBlock,NamFileInput,fname,fInputVar,fGlobalVar,Parallel_info,initialize)
+function [fOutVar,nBlockPerCPU,CPUforIndex,totCPU] = masterParallel(Parallel,fBlock,nBlock,NamFileInput,fname,fInputVar,fGlobalVar,Parallel_info,initialize)
```

Comment:
Sorry Ronal, wouldn't it be better to add the new output argument as the 4th argument?
Keeping totCPU as 3rd output arg would make this extension more backward compatible?

Reply:
bugs associated to this bad output argument list are fixed in e58f85b4b057c5f67cc0725aa667feb87e9a2635